npm/private/test/BUILD.bazel

```diff
@@ -48,3 +48,14 @@ filegroup(
         # intentionally don't include node_modules/unused
     ],
 )
+
+sh_test(
+    name = "bin_test",
+    srcs = ["bin_test.sh"],
+    args = ["$(NODE_PATH)"],
+    data = [
+        ":node_modules/typescript",
+        "//:node_modules/typescript",
```

Comment:
Is this testing something related to having two packages with conflicting bins? Perhaps a clarifying comment?

Reply:
Good call. Added comment.